trustar_connector.py

```diff
@@ -227,8 +230,10 @@ def _make_rest_call_helper(self, endpoint, action_result, headers={}, params=Non
                 retry_failure_flag = True
 
             # If token is expired, generate a new token
-            if error_message and (consts.TRUSTAR_INVALID_TOKEN_MESSAGES[0] in error_message or consts.TRUSTAR_INVALID_TOKEN_MESSAGES[1] in error_message):
-                self.debug_print("Refreshing TRUSTAR API and re-trying request to [{}] because API token was expired or invalid with error [{}]".format(endpoint, error_message))
+            if error_message and (consts.TRUSTAR_INVALID_TOKEN_MESSAGES[0]
+                    in error_message or consts.TRUSTAR_INVALID_TOKEN_MESSAGES[1] in error_message):
```

Comment:
Should we use brackets inside the 2nd part?
```
((consts.TRUSTAR_INVALID_TOKEN_MESSAGES[0] in error_message) or (consts.TRUSTAR_INVALID_TOKEN_MESSAGES[1] in error_message))
```
It might be easier to indent and easier to read later on.

Reply:
Done